### PR TITLE
Bump to RegistryCI v8

### PR DESCRIPTION
**Pull request by ericphanson**

### CODE REVIEW

File: .ci/Project.toml [hunk 1]
```diff
@@ -8,6 +8,6 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 [compat]
 GitHub = "5"
 HTTP = "0.9, 1"
-RegistryCI = "7"
+RegistryCI = "7,8"
```
Comment (ericphanson) [inline .ci/Project.toml:11]: Actually, probably should only allow v8 (need to wait for registration though)
```suggestion
RegistryCI = "8"
```

File: .ci/Project.toml [hunk 2]
```diff
@@ -8,6 +8,6 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 [compat]
 GitHub = "5"
 HTTP = "0.9, 1"
-RegistryCI = "7"
+RegistryCI = "8"
```
Comment (DilumAluthge) [inline .ci/Project.toml:11]:
```suggestion
RegistryCI = "7, 8"
```

**Review by DilumAluthge — CHANGES_REQUESTED** [review]
The order of operations should be:

1. Update the compat entry to include v8, but don't drop v7 from the compat entry.
2. Run the "update manifests" workflow, and merge the resulting PR.
3. Wait a while to make sure all is well. During this time, if something breaks, we can revert the "update manifests" PR.
4. After a while, make a PR to drop v7 from the compat entry.

**Comment by ericphanson** [comment]
Ok, sounds good!

**Review by DilumAluthge — APPROVED** [review]

**Pull request merged**